webapp/src/containers/LiquidityPage/service.tsx

```diff
@@ -218,7 +218,7 @@ export const handleRemovePoolLiquidity = async (
   const list = await getAddressAndAmountListPoolShare(poolID);
   const addressList: any[] = [];
   list.reduce((sumAmount, obj) => {
-    if (sumAmount < Number(amount)) {
```

Comment:
See line 204 of this file: `${Number(amount1).toFixed(8)}@${hash1}`

I do not want to see anything like "Number.toFixed(8)" being passed to an RPC call.

Please change the right places, not the wrong ones thanks.

Reply:
Noted.

Reply:
@monstrobishi If the amount includes more than 8 decimals then the RPC call gives an invalid amount error that's why that check has been applied. If we start passing bignumber then in our RPC-client we need to convert this bignumber to number with toFixed(8). Or are you suggest making it like ${new BigNumber(amount1).toFixed(8)}@${hash1}

Reply:
Yes use BigNumber toFixed

Reply:
And change TypeScript types on all RPC call to take BigNumber, that's the important one

Reply:
As discussed, 
- Will make all RPC calls to accept big numbers and then convert those bignumbers into numbers with toFixed(8) in RPC call itself.
- Will pass new BigNumber(amount1).toFixed(8)}@${hash1} if the amount is accepted with hash in some rpc calls.
- Use bignumber tofixed instead of number toFixed for all amounts

Got your point. Thanks

Reply:
This change is weird, shouldn't it be `sumAmount < amount`, now it just says `if(amount)` which I'm pretty sure is wrong